chanfitness/chaneventstore.go

```diff
@@ -198,13 +207,18 @@ func (c *ChannelEventStore) Start() error {
 		cancel:         cancel,
 	})
 
+	log.Info("ChannelEventStore started")
```

Comment:
nit: could use `defer` instead

Reply:
You mean using defer and putting it at the beginning of the function ? Was not using it because I didn't wanna print it in case we exit in case of an error.

Reply:
but actually undecided which way to go, I find it kind of misleading printing shutdown complete when an error occurred, but open for both approaches.

Reply:
yeah right - then maybe `defer` after the error - no strong opinion really, so it's up to you.

Reply:
ok kept it at the end, because sometimes we have several possibilities for an error to occur.